config/config.exs

```diff
@@ -65,6 +65,24 @@ config :oli,
     String.to_integer(System.get_env("SCREEN_IDLE_TIMEOUT_IN_SECONDS", "1800")),
   log_incomplete_requests: true
 
+config :oli, :dataset_generation,
```

Comment:
Unless these configs are strictly required at compile time or when the release is being built, they can probably solely live in `runtime.exs` to simplify the config and avoid duplication.